doc/nrf/gs_modifying.rst

```diff
@@ -151,8 +221,39 @@ To locate a specific configuration option, use the filter (:guilabel:`Jump to` i
 The documentation for each :ref:`configuration option <configuration_options>` also lists the menu path where the option can be found.
 
 .. important::
-   All changes to the :file:`.config` file are lost when you clean your build directory.
-   You can save it to another location, but you must then manually copy it back to your build directory.
+   All changes to the :file:`.config` file are lost when you clean your build directory or you rebuild the project with the ``pristine`` target.
```

Comment:
This line should be cut.

It is redundant with

"This means that this file is available when building the application, but it is deleted when you clean the build directory."

If the user uses pristine, he should know that it is a very clean build directory. If he didn't know what pristine was he shouldn't have used it.

And it is already documented that cleaning the build directory deletes the .config file.

Reply:
and it's not even correct: https://github.com/nrfconnect/sdk-nrf/pull/6121#discussion_r759119361
> wrong, the .config is kept when you re-run CMake, but will be lost if you remove the build folder / do a pristine build.

clean != remove